GSoC_19/posts/websocket-passive-scan-using-scripts-with-zap.org

```diff
@@ -0,0 +1,845 @@
+#+BEGIN_COMMENT
+.. title: WebSocket Passive Scan using scripts with ZAP
+.. slug: websocket-passive-scan-using-scripts-with-zap
+.. date: 2019-07-11 14:43:55 UTC+03:00
+.. tags: ZAP, tutorial, websockets, passive, scan
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+
+#+END_COMMENT
+#+OPTIONS: toc:3 
+
+* Introduction 
+
+  In this blog post we are going to demonstrate a brand new feature of OWASP
+  Zed Attack Proxy (ZAP). Specifically, we are going to use WebSocket extension in order to
```

Comment:
- `use the`
 - We should use "add-on" instead of "extension" when referring to the whole.